### PR TITLE
Add focus mode

### PR DESCRIPTION
**Pull request by kelloggm**
Also minor formatting improvements

### CODE REVIEW

**Comment by kelloggm** [comment]
@mwillsey I can't assign you to review this so pls add me as an owner/contributor/whatever-er/benevolent assistant dictator/etc. so that I can request a review.

File: crossbot/commands/plot.py [hunk 1]
```diff
@@ -214,6 +223,8 @@ def fmt_min(sec, pos):
             dates, scores = zip(*pairs)
             dates = [datetime.datetime.strptime(d, "%Y-%m-%d").date() for d in dates]
             name = client.user(userid)
+            if not args.focus is None:
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: Use pythonic `is not`.

File: crossbot/commands/plot.py [hunk 2]
```diff
@@ -88,6 +89,14 @@ def init(client):
         ' Ignored if both start-date and end-date given.'
         ' Default %(default)s.')
 
+    focus = parser.add_argument_group('Focus player')
+
+    focus.add_argument(
+        '-f', '--focus',
+        type    = str,
+        metavar = 'F',
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: `metavar` should convey that's it's a username. Maybe `metavar = 'user'`.

File: crossbot/commands/plot.py [hunk 3]
```diff
@@ -222,11 +233,12 @@ def fmt_min(sec, pos):
         colors = [cmap(i / n_users) for i in range(n_users)]
         times = OrderedDict(sorted(times.items()))
         for (userid, entries), color in zip(times.items(), colors):
-
             dates, seconds = zip(*entries)
             max_sec = max(max_sec, max(seconds))
             dates = [datetime.datetime.strptime(d, "%Y-%m-%d").date() for d in dates]
             name = client.user(userid)
+            if not args.focus is None:
```
Comment (mwillsey) [inline crossbot/commands/plot.py]: pythonic `is not`

**Comment by kelloggm** [comment]
@mwillsey I've addressed feedback. Please review again.

**Pull request merged**